app/layout.tsx

```diff
@@ -2,6 +2,15 @@ import React from 'react';
 import './globals.css';
 import type { Metadata } from 'next';
 import clsx from 'clsx';
+import { Poppins } from 'next/font/google';
+
+const poppins = Poppins({
```

Comment:
Is this the best way to do this? Don't we have it already in our tailwind css config?